packages/sdk/README.md

```diff
@@ -2,232 +2,65 @@
+### Authenticate your API calls
 
-Verify authentication works and that API calls will succeed with code similar to the following:
+All requests to the Looker API server require an access token. For browser implementations, authentication is typically achieved via OAuth as [described in `cors.md`](/docs/cors.md) or a [Proxy Server](#using-a-proxy-for-authentication).
 
-```typescript
-import { LookerNodeSDK } from '@looker/sdk-node'
-(async () => {
-  // create a Node SDK object for API 3.1
-  const sdk = LookerNodeSDK.init31()
-  // retrieve your user account to verify correct credentials
-  const me = await sdk.ok(sdk.me(
-    "id, first_name, last_name, display_name, email, personal_space_id, home_space_id, group_ids, role_ids"))
-  console.log({me})
-  // make any other calls to the Looker SDK
-  const dashboards = await sdk.ok(
-    sdk.search_dashboards({title: 'My SDK dashboard'})
-  )
-  if (dashboards.length === 0) {
-    console.log('Dashboard not found')
-  }
-  const [ dashboard ] = dashboards
-  // do stuff with dashboard
+### Use the SDK in your browser application
 
-  ...
+Authenticating for the browser takes more setup than authenticating for use with a Node application.
 
-  await sdk.authSession.logout()
-  if (!sdk.authSession.isAuthenticated()) {
-    console.log('Logout successful')
-  }
-})()
-```
+The stand-alone version of the [Looker API Explorer](/packages/api-explorer) uses OAuth and the `BrowserSDK` to get an authentication token for Looker API requests.
 
-**NOTE**: By default, `LookerNodeSDK.init31()` and `LookerNodeSDK.init40()` will check for environment variables. Environment variables can be ignored by passing an empty string to the NodeSettings constructor.
+- [RunItSDK](/packages/run-it/src/utils/RunItSDK.ts) shows how to override `readConfig()` to get SDK configuration values.
 
-```typescript
-// Ignore any SDK environment variables for the node runtime
-const settings = new NodeSettingsIniFile('')
-const sdk = LookerNodeSDK.init40(settings)
-const sdk31 = LookerNodeSDK.init31(settings)
-```
+- [RunItSDK tests](/packages/run-it/src/utils/RunItSDK.spec.ts) support debugging the flow of `RunItSDK`.
 
-### Using the Browser SDK
+- The [OAuthScene](/packages/run-it/src/scenes/OAuthScene/OAuthScene.tsx) React component receives the OAuth response from the Looker server and logs the user in to retrieve the API authentication token.
 
-The default auth/auth mode for the Browser SDK uses CORS, so your Looker instance will need to be configured for CORS support, which is available in Looker 7.10 and above.
+Looker's OAuth support makes it possible to build a Looker SDK application that only requires the browser. If a browser application can use a proxy server instead, or already uses an existing backend server, it may be simpler to [use a proxy for authentication](#using-a-proxy-for-authentication)/
```

Comment:
Explicitly call out that looker.ini and environment variables will NEVER work in the browser

Reply:
ok